releases/v1.4.2.toml

```diff
@@ -0,0 +1,35 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.4.1"
+
+pre_release = false
+
+preface = """\
+The second patch release for `containerd` 1.4 includes multiple minor fixes
+and updates.
+
+### Notable Updates
+
+* Fix issue on client with container removal when snapshot is deleted [containerd/containerd#3449](https://github.com/containerd/containerd/pull/3449)
```

Comment:
Didn't see this one in https://github.com/containerd/containerd/compare/v1.4.1...release/1.4

Reply:
Updated, there was a bug fix that mentioned that issue but realized it is not related and the commit message is incorrect